sct-app/pom.xml

```diff
@@ -20,6 +20,7 @@
 
     <properties>
         <sonar.coverage.jacoco.xmlReportPaths>${basedir}/${aggregate.report.dir}</sonar.coverage.jacoco.xmlReportPaths>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
The javadoc for sct-app could be activated.

Reply:
done